python/src/usecases/fireSatellite_function.py

```diff
@@ -0,0 +1,511 @@
+"""
+Use case : FireSatellite test function
+======================================
+"""
+import openturns as ot
+import math as m
+
+class FireSatelliteModel():
+    """
+    Data class for the Fire Satellite.
+
+
+	Attributes
+	----------
+	
+	dim : The dimension of the problem
```

Comment:
```suggestion
    dim : The dimension of the problem
```
The white space at the left of `dim` seems to be a tabulation. It would be better to stick to spaces.